package.json

```diff
@@ -30,7 +30,7 @@
     "@dojo/has": "beta1",
     "@dojo/i18n": "beta1",
     "@dojo/shim": "beta1",
-    "maquette": "2.4.3"
+    "maquette": "2.5.0"
```

Comment:
I think for `widget-core` we could make it a dependency.

Sort of a side note, how would `dojo build` resolve multiple versions of `maquette`?  For example if someone has it specified as one version in the `package.json` but includes `widget-core` so you have something like:

```
root
└ node_modules
 ├ @dojo
 │ └ widget-core
 │  └ node_modules
 │    └ maquette@2.5.0
 └ maquette@2.4.3